applications/FluidDynamicsApplication/tests/two_fluid_inlet_test.py

```diff
@@ -0,0 +1,225 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+from fluid_dynamics_analysis import FluidDynamicsAnalysis
+import KratosMultiphysics.kratos_utilities as kratos_utils
+
+try:
+    import KratosMultiphysics.ExternalSolversApplication
+    have_external_solvers = True
+except ImportError as e:
+    have_external_solvers = False
```

Comment:
this entire block is not needed from what I see

Reply:
Removed.